package.json

```diff
@@ -50,6 +50,7 @@
     "electron-updater": "^5.2.1",
     "execa": "^5.1.1",
     "ms": "^2.1.3",
+    "node-stdlib-browser": "^1.2.0",
```

Comment:
Do we understand why the new version of `@glif/filecoin-address` depends on Node stdlib now? It seems like a breaking change to me, maybe we should discuss this with the maintainers?

Reply:
They started requiring the global `Buffer`, which isn't a breaking change in a Node.js context 🤔

Reply:
I'm not sure if adding this whole node stdlib dependency to `@glif/filecoin-address` is worth it, other module consumers might already be using a different stdlib shim. And if multiple modules use different versions, that could lead to bloat. I'm not sure what the proper fix for them would be? Include a `buffer` specific shim? Or require module consumers to include the shim from this PR?